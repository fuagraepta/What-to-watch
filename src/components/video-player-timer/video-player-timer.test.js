import React from 'react';
import renderer from 'react-test-renderer';

import VideoPlayerTimer from './video-player-timer.jsx';

describe(`The component is rendered correctly`, () => {
  it(`Video player timer correctly renders with transferrd mock time`, () => {
    const videoPlayerTimer = renderer
    .create(<VideoPlayerTimer
      seconds = {10}
      minutes = {34}
      hours = {1}
    />)
    .toJSON();

    expect(videoPlayerTimer).toMatchSnapshot();
  });
});
