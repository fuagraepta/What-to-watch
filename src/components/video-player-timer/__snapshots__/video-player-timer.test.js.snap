// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The component is rendered correctly Video player timer correctly renders with transferrd mock time 1`] = `
Array [
  <div
    className="player__time"
  >
    <progress
      className="player__progress"
      max={33}
      value={6}
    />
    <div
      className="player__toggler"
      style={
        Object {
          "left": "18.181818181818183%",
        }
      }
    >
      Toggler
    </div>
  </div>,
  <div
    className="player__time-value"
  >
    00:00:27
  </div>,
]
`;
